include/grpc/grpc_security.h

```diff
@@ -131,6 +131,34 @@ grpc_credentials *grpc_google_iam_credentials_create(
     const char *authorization_token, const char *authority_selector,
     void *reserved);
 
+/* Function to be called by the metadata credentials plugin implementation when
+   the metadata is ready. */
+void grpc_credentials_plugin_metadata_notify(
```

Comment:
naming seems inconsistent with grpc_metadata_credentials_plugin and grpc_metadata_credentials_create_from_plugin. Would any of these be better?
grpc_metadata_credentials_notify()
grpc_metadata_credentials_plugin_notify()  /* that's my favorite */
grpc_metadata_credentials_metadata_notify()